### PR TITLE
new page

### PR DESCRIPTION
**Pull request by Rebal67**

### CODE REVIEW

File: src/pages/New.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+export default function New() {
+    return (
+        <div className="container">
+            <TImage src={require('../assets/tonomy/tonomy-logo1024.png')} height={86}></TImage>
+            <TH1>Tonomy ID</TH1>
+            <TH3>The easiest and safest way to access Tonomy apps</TH3>
+            <TP>
+                Tonomy ID is a self-sovereign identity digital wallet that protects your security and privacy. &nbsp;
+                <TA>Learn more</TA>
```
Comment (theblockstalk) [inline src/pages/New.tsx]: please add link href to config.json files

File: src/App.tsx [hunk 2]
```diff
@@ -1,19 +1,21 @@
 import { Container } from '@mui/material';
 import React from 'react';
 import Home from './pages/Home';
+import New from './pages/New';
```
Comment (theblockstalk) [inline src/App.tsx]: make this clearer name DownloadApp or something

File: src/components/buttons.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.button-link{
```
Comment (theblockstalk) [inline src/components/buttons.css:1]: lets use in component styling whenever possible, makes codebase more modular and easy to reason with

File: src/pages/New.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                It is open-source and maintained by the Tonomy Foundation. <TA>Learn more</TA>
+            </TP>
+            <div>
+                <a href="#">
+                    <img alt="Get it on Apple store" width="200px" src={appStoreImage} />
+                </a>
+            </div>
+
+            <div>
+                <a href="https://play.google.com/store/apps/details?id=foundation.tonomy.projects.tonomyidstaging&hl=en&gl=US&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1">
```
Comment (theblockstalk) [inline src/pages/New.tsx]: also make this href from config.json

File: src/pages/New.tsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+                    <img
+                        alt="Get it on Google Play"
+                        width="230px"
+                        src="https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png"
```
Comment (theblockstalk) [inline src/pages/New.tsx]: lets put this in the src/assets, it scales better (loads faster)

File: src/pages/new.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+.container {
```
Comment (theblockstalk) [inline src/pages/new.css]: in component css, same comment as above

File: types/index.d.ts [hunk 7]
```diff
@@ -0,0 +1,3 @@
+declare module '*.jpg';
```
Comment (theblockstalk) [inline types/index.d.ts:1]: is this necessary?

File: src/App.tsx (hunk 2 above)
Comment (Rebal67) [reply src/App.tsx]: changed

File: src/components/buttons.css (hunk 3 above)
Comment (Rebal67) [reply src/components/buttons.css:1]: i disagree, it's better to use css classes when possible. they are easier to use can be shared between components.
and has better performance.
when we want reactive styles we can still use inline styling.
https://reactjs.org/docs/faq-styling.html

File: types/index.d.ts (hunk 7 above)
Comment (Rebal67) [reply types/index.d.ts:1]: yes it will give type errors when importing images

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**